README.md

```diff
@@ -1,29 +1,29 @@
 # FabricSkyboxes
 
-##### Implements custom skyboxes, similar to Optifine!
+##### Implements Custom Skyboxes, like Optifine! (Not with its format though)
 
-## Support
-
-Please use the issue tracker for bugs.
-
-Alternatively, join my discord server: https://discord.gg/quvzkaV
+![](https://cdn-raw.modrinth.com//data/YBz7DOs8/images/49eab46d1e74e8f781404a8f773a1a09dedda5aa.png)
+_Screenshot taken with the [Awesome Skies](https://github.com/heymanMC/Awesome-Skies) resource pack, a pack made to run **only** under FSB_ (Credit: [heyman](https://github.com/heymanMC))
 
 ## Purpose
 
-This mod allows specification of custom skyboxes, with any textures that you may want.
-You can even specify textures already in Minecraft's resources!
+This mod allows the specification of Custom Skyboxes, with any textures that you may want, even with the ones which are already in Minecraft's resources!
```

Comment:
Probably just remove the bit after the second comma, it doesn't contribute much and isn't helpful information (I don't even know why I thought it was)